fpfilter.pl

```diff
@@ -183,7 +184,8 @@
     if(length($ref) > 1 || length($var) > 1) {
```

Comment:
Correct. However, this PR shouldn't change that behavior. The fpfilter would have classified that as a deletion before.  Suggestions for correctly handling these? My goal with this PR was to simply pass equal length ref/var variants without failing. Perhaps I should reword the commit message and the error message to make that more clear?